i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-dsp-no-sdk.md

```diff
@@ -92,33 +92,37 @@ For an example of code that makes sure the token hasn't expired, see [UID2Encryp
 
 To make sure that the token lifetime has a valid value, check these two conditions:
 
-- The remaining lifetime of the token must not exceed the `max_bidstream_lifetime_seconds` value from the `/v2/key/bidstream` response.
+- The lifetime of the token (in this context, the remaining amount of time for which the current token is still valid) must not exceed the `max_bidstream_lifetime_seconds` value from the `/v2/key/bidstream` response.
```

Comment:
One small correction: the remaining amount of time for which the current token is still valid is only work for v2 token. For later version, the lifetime is the duration for which the token remains valid until it expires. 

Maybe we can remove "(in this context, the remaining amount of time for which the current token is still valid)" because we have provided the calculation method in the block below.

Reply:
Done, thx.